doc/source/quickstart-huggingface.rst

```diff
@@ -6,4 +6,220 @@ Quickstart 🤗 Transformers
+            metric.add_batch(predictions=predictions, references=batch["labels"])
+        loss /= len(testloader.dataset)
+        accuracy = metric.compute()["accuracy"]
+        return loss, accuracy
+
+
+Creating the model itself
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To create the model itself, 
+we will just load the pre-trained distillBERT model using Hugging Face’s :code:`AutoModelForSequenceClassification` :
+
+.. code-block:: python
+
+    from transformers import AutoModelForSequenceClassification 
+
+    net = AutoModelForSequenceClassification.from_pretrained(
+            CHECKPOINT, num_labels=2
+        ).to(DEVICE)
+
+
+Federating the example
+----------------------
+
+Creating the IMDBClient
+^^^^^^^^^^^^^^^^^^^^^^^
+
+To federate our example to multiple clients, 
+we first need to write our Flower client class (inheriting from :code:`flwr.client.NumPyClient`). 
+This is very easy, as our model is a standard :code:`PyTorch` model:
+
+.. code-block:: python
+
+    from collections import OrderedDict
+    import flwr as fl
+
+    class IMDBClient(fl.client.NumPyClient):
+            def get_parameters(self, config):
+                return [val.cpu().numpy() for _, val in net.state_dict().items()]
+            def set_parameters(self, parameters):
+                params_dict = zip(net.state_dict().keys(), parameters)
+                state_dict = OrderedDict({k: torch.Tensor(v) for k, v in params_dict})
+                net.load_state_dict(state_dict, strict=True)
+            def fit(self, parameters, config):
+                self.set_parameters(parameters)
+                print("Training Started...")
+                train(net, trainloader, epochs=1)
+                print("Training Finished.")
+                return self.get_parameters(config={}), len(trainloader), {}
+            def evaluate(self, parameters, config):
+                self.set_parameters(parameters)
+                loss, accuracy = test(net, testloader)
+                return float(loss), len(testloader), {"accuracy": float(accuracy)}
+
+
+The :code:`get_parameters` function lets the server get the client's parameters. 
+Inversely, the :code:`set_parameters` function allows the server to send its parameters to the client. 
+Finally, the :code:`fit` function trains the model locally for the client, 
+and the :code:`evaluate` function tests the model locally and returns the relevant metrics. 
+
+We can now start client instances using:
+
+.. code-block:: python
+
+    fl.client.start_numpy_client(
+        server_address="127.0.0.1:8080", 
+        client=IMDBClient()
+    )
+
+
+Starting the server
+^^^^^^^^^^^^^^^^^^^
+
+Now that we have a way to instantiate clients, we need to create our server in order to aggregate the results. 
+Using Flower, this can be done very easily by first choosing a strategy (here, we are using :code:`FedAvg`, 
+which will define the global weights as the average of all the clients' weights at each round) 
```

Comment:
@charlesbvll I think we should have the start server part before the start client part.